### ISSUE TITLE
Are horses even supported?

### ISSUE THREAD

**Issue by ghost** [PEXPlugins/Modifyworld#99]
I cannot give a player a node that allows them to do anything with horses. Please add a node for horses.

**Comment by chillerkraft** [comment]
i don't think you can specifically use any nodes for 1.8 mobs, but maybe you can do a wildcard and then negate anything above it that u dont want to allow, might be a workaround... hmm... i wonder if modifyworld works at all for 1.8 mobs.... im pretty sure if u have -modifyworld.* then it will even stop vanilla actions with new mobs so u might be able to work something out

PLEASE UPDATE MODIFYWORLD PPLS :yum:
